### PR TITLE
Test new registry key for windows_toolscore (2017)

### PR DESCRIPTION
**Pull request by d3x0r**
re: #100 

Was going to update changelog and package.json versions; but thought that might be too bold.

(as s different issue) I also think -DCMAKE_BUILD_TYPE should be more aggressively set but I haven't identified that yet. (I can just include setting that in my package.json scripts which I think I will do)

v3.6.1 - 11/12/17
=================

- Detect 2017 Windows Build Tools

### CODE REVIEW

**Comment by unbornchikken** [comment]
Very mutch appreciated, thanks. Again. Wanna join as a collaborator? Got too much small kids to keep enough eye on project. Btw, will make a release on this evening, aleast I hope.

File: lib/es6/vsDetect.js [hunk 1]
```diff
@@ -12,11 +12,19 @@ let vsDetect = {
     }),
     _isBuildToolsInstalled: async(function*(version) {
         let mainVer = version.split(".")[0];
-        let key = "HKLM\\SOFTWARE\\Classes\\Installer\\Dependencies\\Microsoft.VS.VisualCppBuildTools_x86_enu,v" + mainVer;
+        let key; 
+        let testPhrase;
+	if( mainVer >= "15" ) {
```
Comment (unbornchikken) [inline lib/es6/vsDetect.js]: That should be (spacing, type handling):

```js
if (Number(mainVer) >= 15) {
```

File: lib/es6/vsDetect.js [hunk 2]
```diff
@@ -12,11 +12,19 @@ let vsDetect = {
     }),
     _isBuildToolsInstalled: async(function*(version) {
         let mainVer = version.split(".")[0];
-        let key = "HKLM\\SOFTWARE\\Classes\\Installer\\Dependencies\\Microsoft.VS.VisualCppBuildTools_x86_enu,v" + mainVer;
+        let key; 
+        let testPhrase;
+	if( mainVer >= "15" ) {
+            key = "HKLM\\SOFTWARE\\Classes\\Installer\\Dependencies\\Microsoft.VS.windows_toolscore,v" + mainVer;
+            testPhrase = "Version";
+        } else {
```
Comment (unbornchikken) [inline lib/es6/vsDetect.js]: else should be on new line

**Comment by d3x0r** [comment]
fixed and pushed (accidentally included changelog and package.json updates)

**Comment by d3x0r** [comment]
Re adding as contributor :) I'm a little scattered, I feel better working through the original maintainer than doing things directly (for example the string comparison and formatting :)  )

**Comment by unbornchikken** [comment]
Oh, I totally forgot this. I'm gonna make it to land on today for sure.

**Comment by unbornchikken** [comment]
Released, ty and sry for the delay.

**Comment by d3x0r** [comment]
No worries :)   Figured someone else would complain with an issue... it's not something I have to reinstall often, so once it was patched I was good :)

**Pull request merged**